api/test/integration/test_rbac_white_all_endpoints.tavern.yaml

```
@@ -2047,13 +2047,27 @@ stages:
       json:
         username: newUser
         password: stringA1!
-        allow_run_as: true
       headers:
         Authorization: "Bearer {test_login_token}"
     response:
       <<: *permission_denied
 
 ---
+test_name: PUT /security/users/{user_id}/run_as
+
+stages:
+  - name: Try to change the allow_run_as field from one user
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/security/users/1/run_as"
+      method: PUT
+      params:
+        allow_run_as: 'false'
```

This parameter should be boolean. The test is not failing as the use of this endpoint is denied.

```
{
  "title": "Bad Request",
  "detail": "Wrong type, expected 'boolean' for query parameter 'allow_run_as'"
}
```

```suggestion
        allow_run_as: false
```

---

You are right. After fix it in my local branch everything keeps working as expected.